src/stash/src/postgres.rs

```diff
@@ -507,15 +496,12 @@ impl Postgres {
     async fn consolidate_batch_tx(
         stmts: &PreparedStatements,
         tx: &Transaction<'_>,
-        collections: Vec<(Id, Option<Antichain<Timestamp>>)>,
+        collections: &[Id],
```

Comment:
The comment above should be updated to remove `Option<current since> ...`

Reply:
Done.